app/src/home.js

```diff
@@ -686,16 +714,23 @@ export default function Home() {
           resultChunks={[...comprehendCustomEntities, ...comprehendResults]}
           excludedItems={excludedItems}
           onToggleItem={toggleResultItemVisibility}
-          visible={stage === STAGE_SUMMARIZE || stage === STAGE_TRANSCRIBING || stage === STAGE_TRANSCRIBED}
+          visible={stage === STAGE_TRANSCRIBING || stage === STAGE_TRANSCRIBED}
           onResultDelete={onComprehendResultDelete}
           onResultAdd={onComprehendResultAddition}
           onSelectedConceptChange={onSelectedConceptChange}
         />
 
+        <SOAPReviewPane
+          visible={stage === STAGE_SOAP_REVIEW}
+          onInputChange={updateSOAPSummary}
+          inputText={soapSummary}
+        />
+
         <ExportPane
           transcriptChunks={transcripts}
           resultChunks={comprehendResults}
```

Comment:
Result chunks need to have both comprehend and custom comprehend entities
See [here](https://github.com/aws-samples/medical-transcription-analysis/pull/58/files#diff-f9907ef5d709c5d6406d453d26a3c4c626be16fe423c616668fc4ad07a6f7034R731)